Sources/UIScrollView.swift

```diff
@@ -77,6 +71,19 @@ open class UIScrollView: UIView {
             addSubview($0)
         }
     }
+    
+    // When adding subviews, make sure that the scroll indicators stay on top
+    open override func addSubview(_ view: UIView) {
+        if view == horizontalScrollIndicator || view == verticalScrollIndicator {
+            super.addSubview(view)
+        } else {
+            super.insertSubview(view, at: subviews.count - 2)
```

Comment:
I understand that this can be concerning. Also don't have anything breakthrough to propose, some small ideas: 
- make indicators their own class so it's more elegant (and safe?)  to check whether something is an indicator or not. 
- insted of inserting at end-2, we could check every element, starting from the last one, for being a scrollIndicator. if it is, we move one back. when it finally isn't, we insert. this should be a little safer.

Reply:
I think we need to avoid hard coding 2 because if as you say the subviews are removed on iOS when hidden we might as well do that

What about the 
insertSubview(below: otherSubview) API?